css/blocks/service/service.less

```
@@ -0,0 +1,29 @@
+.service {
+    background: #EEEEEE;
```

Для цветов, у которых повторяются значения цветов, есть более короткая запись
```suggestion
    background: #EEE;
```
Исправлять не обязательно